### DISCUSSION TITLE
Pain point: License filtering/auditing

### DISCUSSION THREAD

**Discussion by EricStG** [orogene/orogene#33] [Pain Points]
This might be pretty niche, but I've had requirements before preventing packages with certain licenses (ex: GPLv3) or allowing only others, some kind of allow/deny list basically.

Having an easy way to tell which license are currently installed would also be useful.

**Comment by stefee** [comment]
There’s a similar Rust project called cargo-deny (prior art) https://github.com/EmbarkStudios/cargo-deny
